html/modules/custom/reliefweb_user_posts/src/Form/UserPostsPageFilterForm.php

```diff
@@ -30,13 +30,16 @@ public function buildForm(array $form, FormStateInterface $form_state, Moderatio
     // Link to create a new entity.
     $url_options = ['attributes' => ['target' => '_blank']];
 
-    $links = $this->t('Create a new <a href="@job_url">Job vacancy</a> or a new <a href="@training_url">Training program</a>', [
+    $links = $this->t('Create a new <a href="@job_url">Job vacancy</a>, a new <a href="@training_url">Training program</a> or a new <a href="@report_url">Report</a>', [
```

Comment:
This is some conditional checks. Only people with Contributor role should be able to create reports, not everybody.